### ISSUE TITLE
"ValueError: cannot convert float NaN to integer" error for newer generations of NVIDIA GPUs?

### ISSUE THREAD

**Issue by avimanyu786** [HaseloffLab/CellModeller#28]
I'm getting this error on an RTX 3090 card. Since it has also been reported on an RTX 2080 SUPER as mentioned on issue https://github.com/HaseloffLab/CellModeller/issues/26#issuecomment-728847850, could it be an issue for RTX GPUs in particular?

I can run it just fine on a GTX 1050 Ti. But replicating the same thing with an RTX 3090 throws that error whenever I try to load and run an example simulation. The error is also accompanied with an `Aborted (core dumped)` message at random times.

Also, I would like to mention that the 1050 Ti is running on `nvidia-driver-390`, whereas the 3090 is running on `nvidia-driver-460`. I tried to install `nvidia-driver-390` for 3090 to investigate this issue but it keeps giving a black screen giving no other alternative but to switch back to 460 again.

**Comment by avimanyu786** [comment]
The above test was done on Ubuntu 20.04. When I tested this on Ubuntu 18.04, I got the same issue. There is no black screen on the display but the RTX 3090 refuses to work with `nvidia-driver-390` giving only but a maximum resolution of 640 x 480 when I install this driver. The GPU only seems to recognize `nvidia-driver-460` which is the most recent and available version on both OSs. The GTX 1050 Ti discussed above is on Ubuntu 18.04.
